src/NIfTI.jl

```diff
@@ -107,6 +107,17 @@ function byteswap(hdr::NIfTI1Header)
     hdr
 end
 
+"""
+    NIVolume{T<:Number,N,R} <: AbstractArray{T,N}
+An `N`-dimensional NIfTI volume, with raw data of type 
+`R`. Note that if `raw <: Number`, it will be converted to `Float32`. Additionally, the header is automatically
```

Comment:
You're right, I was referring to the behavior of the constructor on line 131. Is it preferable to document constructor behavior in a separate docstring?

Reply:
I think it would be great to document the constructor too. But you can bundle it with the type into a single docstring, just cleanly specify whether you're talking about the type or the constructor.

Sometimes I just document the constructor and leave the type as an undocumented internal detail.